Source/HTTP/WinHttp/winhttp_provider.h

```diff
@@ -46,6 +46,10 @@ struct XPlatSecurityInformation
     XNetworkingSecurityInformation* securityInformation{ nullptr };
 #endif
     uint32_t enabledHttpSecurityProtocolFlags;
```

Comment:
can you change this line:
    uint32_t enabledHttpSecurityProtocolFlags;
to:
    uint32_t enabledHttpSecurityProtocolFlags{ 0 };
for extra safety